models/models.py

```diff
@@ -209,10 +209,10 @@ def ganet(weights = None,  is_train=False, device='cuda'):
     return model
 
 
-def yolov8(weights=None):
+def yolov8(weights=None, device='cpu'):
     return YOLOv8Builder(
         input_block=Block(1, 1),
-        backbone=YOLO(weights),
+        backbone=YOLO(weights).to(device),
```

Comment:
Are you sure this transfer to the device is needed here? (yolov8 is also used by me)